src/main/webapp/script.js

```diff
@@ -14,3 +14,10 @@
 
 
 
+function userLogin() {
+  fetch('/login').then(response => response.text())
+  .then((message) => {
```

Comment:
at some point, consider adding error handling.

what happens if the server times out?